clouddq/lib.py

```diff
@@ -151,6 +162,7 @@ def create_entity_summary_model(
         "entity_target_rule_binding_configs": entity_target_rule_binding_configs,
         "gcp_project_id": gcp_project_id,
         "gcp_bq_dataset_id": gcp_bq_dataset_id,
+        "rule_binding_configs": dq_target_rule_binding_configs_dict,
```

Comment:
please rename both "rule_binding_configs" and "dq_target_rule_binding_configs_dict" to "failed_queries_configs"